src/Csharp/packages/raytracer/src/RayTracerNative.cs

```diff
@@ -16,7 +16,7 @@ namespace DHARTAPI.RayTracing
 {
 
 	/*! \param NativeMethods for the RayTracing namespace */
-	internal static class NativeMethods
+	internal sealed class NativeMethods
```

Comment:
Why is this changed to `sealed`?

Reply:
Performance reasons. Unless you want to derive from a class, it's generally a good idea to seal classes in c#
https://www.youtube.com/watch?v=d76WWAD99Yo

Reply:
I get the point he is making against regular classes, but we are using `static`. 

https://social.technet.microsoft.com/wiki/contents/articles/21028.difference-between-static-class-sealed-class-and-abstract-class-in-c.aspx

**Static Class:** ... This class **cannot be instantiated,** i.e we cannot have objects of this class. To access methods of this class, you can directly use classname.method. Also this class cannot be inherited.

**Sealed Class**:.... this class cannot be inherited. But **we can instantiate this class**, i.e we can have any number of objects of a sealed class.

So from the API standpoint, is there a reason you need to instantiate the class? I think this might be dangerous in the way we are interfacing with C, but i'm not sure. 


https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/static-classes-and-static-class-members

> Static classes are sealed and therefore cannot be inherited.

Reply:
Good point!
I’ll revert these back to static.